README.md

```diff
@@ -43,7 +43,7 @@ are several ways to speed up or otherwise improve on this in the future.
 1. `git clone` this repo into a convinient place ( e.g `~/hackman` )
 1. cd into the repo
 1. `ansible-playbook ansible-installer.yml`
-1. If replacing an old system, restore the old database to `/var/www/hackman/db/db.sqlite3`
+1. If replacing an old system, restore the old database to `/var/www/hackman/db.sqlite3`
```

Comment:
The database moved with the new build tools - if the documented process here is followed then the /var/www/hackman/db/db.sqlite3 is the correct directory.

If the years old legacy installation currently used in the space is referenced, the database directory is different - but the legacy system is probably not the best to document